react/components/InfoCard/index.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent } from 'react'
+import { bool, string, oneOf } from 'prop-types'
+import classNames from 'classnames'
+import { withRuntimeContext } from 'vtex.render-runtime'
```

Comment:
If you change to a functional component, you could try `useRuntime` hook. Your tree would look cleaner